demo-projects/relationships/README.md

```diff
@@ -0,0 +1,18 @@
+# Demo Project: Relationships
+
+This project demonstrates how to set up different types of relationships with Keystone.
+It is particularly useful if you want to examine how different relationship types are stored in the database.
+
+## Running the Project.
+
+To run this demo project, all you need to do is run `yarn` within the Keystone project root to install all required packages, then run `REL_TYPE=one_one_to_many yarn manypkg run demo-projects/relationships dev` to begin running Keystone.
```

Comment:
I think this command will only work on MacOS: 
```
REL_TYPE=one_one_to_many yarn manypkg run demo-projects/relationships dev
```
This also need to be run from the root of the monorepo? And... I'm not sure the `manypkg` part is required?

Reply:
I'd modify the start command in package.json, use `crossenv` so those variables work... then I think the command would simply be `yarn demo relationships`

Reply:
Or `yarn demo relationships one_one_to_many` 

one_one ???